models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -115,7 +167,10 @@ kyberswap_dex AS (
 SELECT
     'avalanche_c'                                                         AS blockchain
     ,'kyberswap'                                                          AS project
-    ,'dmm'                                                                AS version
+    ,kyberswap_dex.version                                                AS version
+    ,SELECT (CASE
+    WHEN (kyberswap_dev.version = 'dmm' or kyberswap_dev.version = 'elastic' ) then 'dex'
+    ELSE 'aggregator' END)                                                AS category
```

Comment:
Here is a little typo.
```suggestion
    ,CASE
        WHEN (kyberswap_dex.version = 'dmm' or kyberswap_dex.version = 'elastic' ) THEN 'dex'
        ELSE 'aggregator'
    END                                                                   AS category
```